### PR TITLE
Update Setup Credentials on README.md

### PR DESCRIPTION
**Pull request by irfano**
This adds information to README.md to indicate what should use for Website URL and Javascript Origins fields on the applications manager.

This was [asked](https://wordpress.slack.com/archives/C02RQC4LY/p1666235077945229) for WPAndroid in WordPress Slack channel.

The new sentence is copied from [WCAndroid documentation](https://github.com/woocommerce/woocommerce-android/blob/trunk/docs/project-overview.md#oauth2-authentication).

To test:
1. Check [README.md](https://github.com/wordpress-mobile/WordPress-iOS/blob/47e1c5a952e8d45846fe6b4431475dcaeacbb957/README.md) if the change is correct.

## Regression Notes
1. Potential unintended areas of impact
None.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Nothing.

3. What automated tests I added (or what prevented me from doing so)
None. This is a document update.

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,7 +83,8 @@ In order to login to WordPress.com with the app you need to create an account ov
 
 After you create an account you can create an application on the [WordPress.com applications manager](https://developer.wordpress.com/apps/).
 
-When creating your application, select "Native client" for the application type. The applications manager currently requires a "redirect URL", but this isn't used for mobile apps. Just use "https://localhost".
+When creating your application, you should select "Native client" for the application type.
+The "**Website URL**", "**Redirect URLs**", and "**Javascript Origins**" fields are required but not used for the mobile apps. Just use "**[https://localhost](https://localhost)**".
```
Comment (mokagio) [inline README.md]: Nitpick: Does the localhost suggestion need to be a link? 

```suggestion
The "**Website URL**", "**Redirect URLs**", and "**Javascript Origins**" fields are required but not used for the mobile apps. Just use "**`https://localhost`**".
```

It's fine either way. Totally up to you 🤷‍♂️

**Review by mokagio — APPROVED** [review]
Thank you for taking the time to improve this documentation 😄

**Review by staskus — APPROVED** [review]
Thanks, @irfano! 👍

File: README.md (hunk 1 above)
Comment (irfano) [reply README.md]: Good point! No need to cause unnecessary clicks. 🙂
Fixed → a515cefad63c434d3e6a3be29c9a9cda13853eb1

**Comment by mokagio** [comment]
@irfano merge at will 😄

**Pull request merged**